packages/harmony/src/icons/IconDetails.stories.mdx

```diff
@@ -0,0 +1,145 @@
+import { Meta, Title } from '@storybook/blocks'
+
+import { Flex } from 'components'
+import { IconQuestionCircle, IconStars } from 'icons'
+import addIconArrangement from 'storybook/assets/addIconArrangement.png'
+import exportIconCommand from 'storybook/assets/exportIconCommand.png'
+import exportIconLayer from 'storybook/assets/exportIconLayer.png'
+import verifyIconSettings from 'storybook/assets/verifyIconSettings.png'
+
+<Meta title='Icons/Details' />
+
+<Title>Details</Title>
+
+### Overview
+
+An icon is a graphic symbol designed to visually indicate the purpose of an interface element.
+
+export function Step(props) {
+  const { index, children } = props
+  return (
+    <section
+      css={(theme) => ({
+        background: theme.color.background.surface1,
+        borderRadius: theme.cornerRadius.s,
+        border: `1px solid ${theme.color.border.strong}`,
+        padding: theme.spacing.l,
+        marginTop: theme.spacing['2xl']
+      })}
+    >
+      {index === undefined ? null : <h2>Step {index}</h2>}
+      {children}
+    </section>
+  )
+}
+
+# For Developers
```

Comment:
Since it's for developers, do we want a section or callout here that links to the Icon component docs?